start-site/src/main/resources/application.yml

```diff
@@ -51,8 +51,8 @@ initializr:
         artifactId: spring-boot-admin-dependencies
         versionProperty: spring-boot-admin.version
         mappings:
-          - compatibilityRange: "[2.3.0.M1,2.5.0-M1)"
```

Comment:
Shouldn't we keep the `2.3.x` version for Spring Boot < 2.4.x ?

Reply:
Hi @snicoll! I have talked to the development team and we agree that the version range I have defined in this PR is correct since Spring Boot Admin is downwards compatible. :)

Reply:
Fair enough.